docs/application/dotnet/guides/nui/layer.md

```diff
@@ -39,12 +51,41 @@ layer.Add(_imageView);
 
 ### Re-ordering layers
 
-A range of functions are provided to change the draw order of the layers.
+By default root layer obtained from the Window instance has a *Depth* field value equal to 0, the same as a newly created layer. Only adding a layer to the window using api *window.AddLayer()* set *Depth* value as it is shown on code example below
+
+```
+Window window = Window.Instance;
+//window.GetDefaultLayer().Depth = 0;
+
+Layer ly0 = new Layer();
+//ly0.Depth = 0
+window.AddLayer(ly0);
+//ly0.Depth = 1;
+
+Layer ly1 = new Layer();
+//ly1.Depth = 0;
+window.AddLayer(ly1);
+//ly1.Depth = 2;
+```
+
+Renderer draws layers and its content from the layers with lowest depth at the beginning. In result a layer and its content with higher depth value will be drawn at the top of the stack.
+
+To reorder layers a set of API listed below can be used: 
+ - Lower(): decrements *Depth* parameter
+ - LowerToBottom(): sets *Depth* parameter to 0. It reorders other layers, and increments root layer *Depth*
+ - MoveAbove(Layer l): moves a layer *Depth* directly above the giver layer
+ - MoveBelow(Layer l): similar to *MoveAbove()* 
+ - Raise(): increments a layer *Depth*
+ - RaiseToTop(): move layer to the top of the layers stack. 
```

Comment:
Just a question (Layer 1) is an argument or what? Do we write API names like this ? 
Kindly verify, if I am not correct, kindly ignore.

Reply:
- Layer 1 : How about changing it as like \<layer name> ?
-  similar to \*MoveAbove()\*  : I think it would be better to change like below.

```
- MoveBelow(<layer name>): moves a layer **Depth** directly below the giver layer
```